packages/protocol/test/common/multisig.ts

```diff
@@ -40,6 +40,32 @@ contract('MultiSig', (accounts: any) => {
     })
   })
 
+  describe('fallback function', () => {
```

Comment:
zeppelin suggested emulating multisig event emission for consistency, added tests to the new case and went in and added those same tests for existing multisig functionality